core/src/com/po/fuck/GeometryMisc.java

```diff
@@ -36,6 +37,18 @@ public static <A extends Entity, B extends Entity> Vector2 direction(A from, B t
         return to.position.cpy().sub(from.position);
     }
 
+    public static <A extends Entity> Entity findClosest(A entity, List<A> list) {
+        if (list.isEmpty()) return null;
+        Entity target = list.get(0);
+        float dist = GeometryMisc.distance(entity, target);
+        for (Entity opponent : list) {
+            if (GeometryMisc.distance(entity, opponent) < dist) {
```

Comment:
It is allowed to use functions from its class without mentioning the class. Thus, `GeometryMisc.distance()` can be replaced with `distance()`